README.md

```diff
@@ -9,6 +9,23 @@
 junit-quickcheck is a library that supports writing and running property-based
 tests in JUnit, inspired by QuickCheck for Haskell.
 
+Property-based tests capture characteristics, or "properties", of the output
+of code that should be true given arbitrary inputs that meet certain criteria.
+For example, imagine a function that produces a list of the prime factors of
+a positive integer `n` greater than 1. Regardless of the specific value of
+`n`, the function must give a list whose members are all primes, must
+equal `n` when all multiplied together, and must be different from the
+factorization of a positive integer `m` greater than 1 and not equal to
+`n`.
+
+Rather than testing such properties for all possible inputs, junit-quickcheck
+and other QuickCheck kin generate some number of random inputs, and verify
```

Comment:
When junit-quickcheck finds a counterexample that disproves a property, it prints the counterexample and also the seed for the randomness generator for each parameter of the theory. This should help when re-running the property test to verify against the counterexample.

See [Seed](http://pholser.github.io/junit-quickcheck/site/0.8-alpha-6/usage/seed.html) for more details.